jdenticon/jdenticon-tests.ts

```diff
@@ -0,0 +1,11 @@
+import {jdenticon} from "./index";
```

Comment:
No, I don't think so.  I'm sorry for the blunder.  I'll make a new commit immediately.

Also, I'm sorry about the delay, but I didn't see your comment until now.